app/labor/badge_rewarder.rb

```diff
@@ -1,8 +1,19 @@
 module BadgeRewarder
+  YEARS = { 1 => "one", 2 => "two", 3 => "three", 4 => "four", 5 => "five", 6 => "six", 7 => "seven", 8 => "eight", 9 => "nine", 10 => "ten" }.freeze
+
   def self.award_yearly_club_badges
-    award_one_year_badges
-    award_two_year_badges
-    award_three_year_badges
+    (1..3).each do |i|
```

Comment:
Not sure how the DEV team would like this, to keep it easy for future updates. I don't like the idea of having it "auto update" based on current year.

Reply:
I agree. Let's not have it auto-update yet.

Reply:
@maestromac So the current state is good to remove WIP? Or should `(1..3).each` be changed to an environment variable?
Will need to figure out how to do that, but I think I have a basic idea how.

Reply:
Yep you can remove the WIP. It's fine to keep it as is.